src/content/en/fundamentals/performance/optimizing-javascript/code-splitting/index.md

```diff
@@ -0,0 +1,745 @@
+project_path: /web/fundamentals/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: TBD
```

Comment:
maybe, "Modern sites often contain a lot of JavaScript. These scripts are often sent down in large, monolithic bundles which can take a long time to download and process. Code-splitting encourages breaking up these scripts so you only send what a user needs when they need it"?